changelog.d/8879.misc

```diff
@@ -0,0 +1 @@
+Pass `room_id` to `get_auth_chain_difference`.
```

Comment:
Should this be what the eventual change will be instead (i.e. should it match #8868)? We seem to end up with really large changelogs, much of which are refactoring to support a feature / fix a bug.

Reply:
Yeah, though I think that is best done by changing the changelog when that PR lands, rather than risking this landing and going in a release before the other PR lands, iyswim

Reply:
Makes sense!